sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIUtils.java

```diff
@@ -28,6 +29,16 @@ public static void addAzureVersionToRequestOptions(String endpoint, RequestOptio
         }
     }
 
+    /**
+     * Add the version query parameter to the request options if the service is not Azure OpenAI service.
+     */
+    public static void addAzureVersionToRequestOptions(String endpoint, RequestOptions requestOptions,
```

Comment:
[Conversation](https://github.com/Azure/azure-sdk-for-java-pr/pull/1434#discussion_r1989292505) continues here:

Both `OpenAIServiceVersion` and `AzureResponsesServiceVersion` implement `ServiceVersion`, and we call getVersion() to get the String value. So, it is possible to be merge both helper methods.